smdebug/pytorch/hook.py

```diff
@@ -118,6 +129,47 @@ def _prepare_collections(self):
 
     # This hook is invoked by trainer prior to running the forward pass.
     def forward_pre_hook(self, module, inputs):
+        if self.profiler_enabled and self.forward_start != 0:
+            self.timeline_writer.write_trace_events(
+               op_name="forward",
+               #timestamp=time.time(),
+               duration=self.forward_end-self.forward_start,
+               step_num=self.step)
+            self.timeline_writer.write_trace_events(
+               op_name="backward",
+               #timestamp=time.time(),
+               duration=self.backward_end-self.backward_start,
+               step_num=self.step)
+            self.timeline_writer.flush()
```

Comment:
writer is flushed in close_Writers(). is it required here?

Reply:
I removed the function call